### PR TITLE
chore: move deploy to circleci

### PR DESCRIPTION
**Pull request by olizilla**
see: https://github.com/ipfs-shipyard/ipfs-dns-deploy

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Comment by olizilla** [comment]
oh me oh my. that was harder than I thought it'd be

**Comment by olizilla** [comment]
Merging for glorious continuous deployment future

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - run:
+          command: make build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - public
+
+  deploy:
+    docker:
+      - image: olizilla/ipfs-dns-deploy:1.1
```
Comment (Mr0grog) [inline .circleci/config.yml]: @olizilla this seems neat and potentially super useful. Is the source code for it available anywhere? (It’s not linked from Docker Hub, unless I’m missing something.)

Comment (olizilla) [reply .circleci/config.yml]: @Mr0grog for sure! https://github.com/ipfs-shipyard/ipfs-dns-deploy

Comment (Mr0grog) [reply .circleci/config.yml]: Ooh, that’s even fancier than I’d realized! Nice 👌

**Pull request merged**